tests/examples/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+if(BUILD_EXAMPLES AND BUILD_TESTING)
+    set(EXAMPLE_BINARY_DIR ${CMAKE_BINARY_DIR}/examples)
+    add_test(
+	NAME "ExampleTesting"
+	COMMAND sh -c "MOCOCRW_EXAMPLE_BINARY_PATH=${EXAMPLE_BINARY_DIR} MOCOCRW_SRC_DIR=${EXAMPLE_DIR} /usr/bin/python3 -m unittest"
```

This is subject to word splitting if you pass it to `sh -c`, so you should either quote or pass things as variables:

```suggestion
	COMMAND sh -c "MOCOCRW_EXAMPLE_BINARY_PATH='${EXAMPLE_BINARY_DIR}' MOCOCRW_SRC_DIR='${EXAMPLE_DIR}' /usr/bin/python3 -m unittest"
```

or

```suggestion
	COMMAND sh -c "env MOCOCRW_EXAMPLE_BINARY_PATH=$1 MOCOCRW_SRC_DIR=$2 /usr/bin/python3 -m unittest" -- "${EXAMPLE_BINARY_DIR}" "${EXAMPLE_DIR}"
```

Also, why do you hardcode `/usr/bin/python3` instead of relying on `$PATH` to locate the correct python3?